Gopkg.toml

```diff
@@ -101,3 +101,7 @@
   go-tests = true
   non-go = true
   unused-packages = true
+
+[[constraint]]
+  name = "github.com/emicklei/go-restful"
```

Comment:
why do we need go-restfull lib?

Reply:
just for getting signature. the code uses go-restful lib.